README.md

```diff
@@ -1,24 +1,148 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# OOP school library
```

Comment:
[Highly Recommended]
- There is a mistake here. Please fix the title, it will be something like `Blog App`. As it is a small mistake, I'm approving the project but please fix this before merging.

Reply:
Hey Dipeshtwis, Thank you for your review 🙏🏻 
I have fixed the README mistake required